tools/spaTyper/spaTyper.xml

```diff
@@ -0,0 +1,60 @@
+<tool id="spaTyper" name="Generate spa type identification" version="0.1.0+galaxy0" python_template_version="3.5" profile="21.05">
```

Comment:
Also, as another comment, it's best to keep the `version` here in-sync with the `version` of the underlying conda package (if this wraps a single Galaxy tool). One nice way to do this is to wrap up the version in a variable like so.

1. Create macro with `@VERSION@` variable:

   ```xml
    <macros>
        <token name="@VERSION@">0.2.1</token>
    </macros>
   ```
   
2. Use `@VERSION@` in place of specifying the version number of the tool. That is change `version="0.1.0+galaxy0"` to `version="@VERSION@+galaxy0"`.

3. Use `@VERSION@` in place of the package version in the requirements. That is replace `version="0.2.1"` with `version="@VERSION@"`.

Please see this bit of code in staramr:

https://github.com/phac-nml/galaxy_tools/blob/c32ec4a9284ec939d001bd7d7d551c15feaa5333/tools/staramr/staramr_search.xml#L1-L12

Reply:
Thanks for this Aaron! Great to have your experience on this